.github/workflows/pr-md-link-check.yaml

```diff
@@ -0,0 +1,23 @@
+name: PR check Markdown links
```

Comment:
The link check actions aren't that useful in CAPV as there's not nearly as many markdown files, but I still think it's alright to include this to prevent link rot over time.